app/ts/components/RecentTransfers.tsx

```diff
@@ -0,0 +1,28 @@
+import { useComputed } from '@preact/signals'
+import { useRecentTransfers } from '../store/recent-transfers.js'
+
+export const RecentTransfers = () => {
+	const { recentTxns } = useRecentTransfers()
+
+	const sortedTxns = useComputed(() => recentTxns.value.sort((a, b) => new Date(b.date).getTime() - new Date(a.date).getTime()))
```

Comment:
Just realized that i had to sort because i am pushing at the end of the array, created a separate issue to rework the adding of transaction method #120